index.html

```diff
@@ -2159,87 +2120,53 @@ <h3>Consumption</h3>
         </p>
 
       </section>
-    </section>
-
-    <section>
-      <h2>JSON-LD</h2>
-
-      <p>
-JSON-LD [[JSON-LD11]] is a JSON-based format used to serialize <a
-href="http://www.w3.org/TR/ld-glossary/#linked-data">Linked Data</a>. This
-section defines the [=production=] and [=consumption=] rules for the
-JSON-LD [=representation=].
-      </p>
-
-      <p>
-The JSON-LD [=representation=] defines the following
-[=representation-specific entries=]:
-      </p>
-
-      <dl>
-        <dt><dfn>@context</dfn></dt>
-        <dd>
-The <a href="https://www.w3.org/TR/json-ld11/#the-context">JSON-LD Context</a>
-is either a <a data-cite="INFRA#string">string</a> or a <a
-data-cite="INFRA#list">list</a> containing any combination of <a
-data-cite="INFRA#string">strings</a> and/or <a data-cite="INFRA#maps">ordered
-maps</a>.
-        </dd>
-      </dl>
 
       <section>
-        <h3>Production</h3>
+        <h2>Generalized JSON-LD Processors</h2>
```

Comment:
As discussed during today's meeting, I find the term "**generalized** JSON-LD processors" confusing in this context. It seems to imply that those are processors *more general* than standard JSON-LD processors, which (IIUC) is *not* what is intended.

From @msporny 's explanations during today's meeting, I think that "general JSON-LD processors" (a wording that is used twice in the paragraph!) or "generic JSON-LD processors" would better convey the intended meaning.

```suggestion
        <h2>General JSON-LD Processors</h2>
```

Reply:
I would have a preference to use the term "Generic" (if we really want to qualify it). That term suggests using the processor as defined (presumably in a standard) whereas the term "General" might mean that there is also a "Specialized" JSON-LD Processor. And that is not the case; there are JSON processors that do not try to interpret `@context` but just consider it as a bona fide JSON key, but that does not qualify to refer to them as JSON-LD processors in the first place.

Reply:
"General" implies contrasting "Special".
"Generic" implies contrasting "Specific".
"Generalized" implies contrasting "Specialized".

Reply:
I ended up dropping "Generic/General/Generalized" and just said "standard JSON-LD Processing algorithms" or "JSON-LD Processing using standards-compliant libraries" since it felt cleaner. Hope that works for everyone.